### PR TITLE
Nhanh 2

### PR DESCRIPTION
**Pull request by thetai95**

### CODE REVIEW

File: Models/Category.php [hunk 1]
```diff
@@ -0,0 +1 @@
+<h1> dong 1 </h1>
```
Comment (thetaiframgia) [inline Models/Category.php]: change 1 file nua

Comment (thetai95) [reply Models/Category.php]: Ok

File: Models/Category.php [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+
+<h1> dong 2 </h1>
```
Comment (thetai95) [inline Models/Category.php]: change them dong nua

Comment (thetaiframgia) [reply Models/Category.php]: deo cho change

**Pull request merged**